bulk_query_gen/influxdb/influx_iot_common.go

```diff
@@ -76,3 +76,28 @@ func (d *InfluxIot) averageTemperatureDayByHourNHomes(qi bulkQuerygen.Query, nHo
 	q := qi.(*bulkQuerygen.HTTPQuery)
 	d.getHttpQuery(humanLabel, interval.StartString(), query, q)
 }
+
+func (d *InfluxIot) IotAggregateKeep(qi bulkQuerygen.Query) {
+	interval := d.AllInterval.RandWindow(7 * 24 * time.Hour)
```

Comment:
We might want to make this window size configurable via a flag at some point, but I think it's ok for now.